formats/iso4.go

```diff
@@ -25,6 +25,16 @@ func NewISO4(cipher Cipher) *ISO4 {
 	}
 }
 
+// Format returns iso type
+func (i *ISO4) Format() string {
+	return "Format 4 (ISO-4)"
+}
+
+// Padding returns padding pattern
+func (i *ISO4) Padding(pin string) string {
+	return strings.Repeat(i.Filler, 16-len(pin)-2)
```

Comment:
`strings.Repeat` can crash if given a negative value. Can we check the math does not result in a negative value?

Example: https://go.dev/play/p/WqNN7XGXiz- 

![Screenshot 2023-06-15 at 10 37 44 AM](https://github.com/moov-io/pinblock/assets/120951/a2c0ea14-bf1b-4256-8dfb-f82c6b3f4da6)

Reply:
We check pin length [before calling](https://github.com/moov-io/pinblock/pull/6/files#diff-b031b231c27395774d31b2d0a6383c1fb8f3d46df1214dc9a865820b4a5fdf73R41) the `Padding`, but I think as we now call it separately, we may move validation logic inside and it will return both padding and error